runsc/boot/loader.go

```diff
@@ -319,6 +322,17 @@ func New(args Args) (*Loader, error) {
 		return nil, fmt.Errorf("creating pod mount hints: %v", err)
 	}
 
+	var stdioFDs []int
+	newfd := startingStdioFD
+	for _, fd := range args.StdioFDs {
+		err := syscall.Dup3(fd, newfd, 0)
```

Comment:
Please set O_CLOEXEC on the new FD to make sure it doesn't "leak" on exec.

Reply:
Will do, thanks for pointing this out.